parser/sql/statement/type/mysql/src/main/java/org/apache/shardingsphere/sql/parser/statement/mysql/dml/MySQLDeleteStatement.java

```diff
@@ -38,6 +39,8 @@ public final class MySQLDeleteStatement extends DeleteStatement implements MySQL
     
     private ReturningSegment returningSegment;
     
+    private WithSegment withSegment;
```

Comment:
Can you provide the documentation for MySQL DELETE WITH ?

Reply:
For this class, you change the SQL parsing result, you need to add SQL Parser IT test to ensure that we handle withSegment

Reply:
Will make these changes once the logic for the CTE's Unique Alias Name is clear.